test/es-module/test-esm-encoded-path-native.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const common = require('../common');
+const common = require('../common/fixtures');
```

Comment:
Did this test ever work? It seems like it did not? `common` itself should still be required as it is going to load a couple of other things as well. The return value here should also be renamed to fixtures, since it is not `common` that is returned here anymore.